.github/workflows/test.yml

```diff
@@ -0,0 +1,68 @@
+name: Run API tests
+on:
+  push:
+permissions:
+  id-token: write
+  contents: write
+  actions: write
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    environment: intg
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: coursier/cache-action@v6
+      - name: Configure AWS credentials from management account
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::${{ secrets.MANAGEMENT_ACCOUNT }}:role/GithubActionsRoleMgmt
```

Comment:
Shouldn't our IAM role names be prefixed with `TDR`, ie `TDRGithubActionsRoleMgmt`

Reply:
Yeah, I've updated the terraform-environments and terraform-backend roles to be prefixed with TDR